roadmap.yaml

```diff
@@ -165,17 +170,29 @@ Data Engineering:
       - Amazon Kinesis
   Pipeline Orchestration:
     - Apache Airflow
+    - Prefect
+    - Dagster
     - Mage
     - Argo
   Messaging and Streaming Platform:
     - Apache Kafka
+    - Connector:
+        - Kafka Connect
```

Comment:
Is connector a common term/concept in messaging and streaming platforms?

Reply:
Yes connector used for connect 2 platforms together and here I mean Kafka connect

Reply:
How about the other tools like NATS and Pulsar? Is it still in common?

Reply:
and apply this one to another PR, please!

Reply:
Done